packages/stark-build/config/webpack.common.js

```diff
@@ -139,6 +138,13 @@ module.exports = options => {
 		 */
 		module: {
 			rules: [
+				{
+					// Mark files inside `@angular/core` as using SystemJS style dynamic imports.
```

Comment:
We should add the reference to the Github issue (https://github.com/angular/angular/issues/21560) so we understand where this come from